mix.exs

```diff
@@ -24,47 +24,39 @@ defmodule Scenic.Mixfile do
         # source_url: "https://github.com/boydm/scenic",
         # homepage_url: "http://kry10.com",
       ],
-        package: [
+      package: [
         name: :scenic,
         contributors: ["Boyd Multerer"],
         maintainers: ["Boyd Multerer"],
         licenses: ["Apache 2"],
         links: %{github: @github}
       ],
-      dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:mix, :iex, :scenic_math]]
+      dialyzer: [plt_add_deps: :transitive, plt_add_apps: [:mix, :iex, :scenic_math]],
+      test_coverage: [tool: ExCoveralls],
+      preferred_cli_env: [
+        coveralls: :test,
+        "coveralls.html": :test,
+        "coveralls.json": :test
+      ]
     ]
   end
 
-  # Configuration for the OTP application
-  #
-  # Type "mix help compile.app" for more information
   def application do
     [
       # mod: {Scenic, []},
       applications: [:logger]
     ]
   end
 
-  # Dependencies can be Hex packages:
-  #
-  #   {:mydep, "~> 0.3.0"}
-  #
-  # Or git/path repositories:
-  #
-  #   {:mydep, git: "https://github.com/elixir-lang/mydep.git", tag: "0.1.0"}
-  #
-  # To depend on another app inside the umbrella:
-  #
-  #   {:myapp, in_umbrella: true}
-  #
-  # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:scenic_math, "~> #{@version}"},
+      {:scenic_math, "~> 0.7.0"},
```

Comment:
@boydm the reason I did this is because some tools can analyze this statically

Reply:
Looking at all this now.

I'd rather it be {:scenic_math, "~> 0.7"}, so it automatically gets the most recent minor version.